pkg/connector/user.go

```diff
@@ -305,7 +306,8 @@ func (u *userResourceType) List(ctx context.Context, _ *v2.ResourceId, pt *pagin
 
 		ur, err := userResource(ctx, userEntry)
 		if err != nil {
-			return nil, pageToken, nil, err
+			l.Warn("baton-ipa: failed to create user resource", zap.String("dn", userEntry.DN), zap.Error(err))
```

Comment:
Do we know what ldap entries we're failing on? It seems like we should make userResource more robust and/or filter out certain entries. This could cause us to silently fail to sync some data.